### PR TITLE
Update the create database examples to use synchronous v2

### PR DESCRIPTION
**Pull request by NHDaly**
Fixes https://github.com/RelationalAI/rai-sdk-julia/issues/59.

### CODE REVIEW

File: src/results.jl [hunk 1]
```diff
@@ -82,7 +82,8 @@ function show_result(io::IO, rsp::TransactionResponse)
 
     for index in eachindex(rsp.metadata)
         println(io, rsp.metadata[index]["relationId"])
-        tuples = zip(rsp.results[index][2]...)
+        data = rsp.results[index][2]
+        tuples = isempty(data) ? [()] : zip(data...)
```
Comment (NHDaly) [inline src/results.jl:86]: Oops, this was an unrelated fix that snuck in here; i didn't mean to commit it. Lemme back that out and do it separately

File: examples/clone-database.jl [hunk 2]
```diff
@@ -22,25 +22,21 @@ include("parseargs.jl")
...
-function run(database, engine, source; profile)
+function run(database, source; profile)
     conf = load_config(; profile = profile)
     ctx = Context(conf)
-    rsp = create_database(ctx, database, engine; source = source)
-    while !is_term_state(get(rsp, "state", ""))  # wait for terminal state
-        sleep(3)
-        rsp = get_database(ctx, database)
-    end
+    # NOTE: create_database is a synchronous operation, so there is no need to poll.
```
Comment (ary) [inline examples/clone-database.jl:28]: Do we need the note? If there are other API methods that are asynchronous, then it definitely makes sense to call this out explicitly. If not, then I think omitting this reduces confusion. Same for `create_database()`

File: examples/create-database.jl [hunk 3]
```diff
@@ -12,34 +12,29 @@
 # See the License for the specific language governing permissions and
 # limitations under the License
 
-# Create a database, optionally overwriting an existing database.
+# Create a database, and print the response
```
Comment (ary) [inline examples/create-database.jl:15]: Nit alert :D

```suggestion
# Create a database and print the response.
```

**Review by ary — APPROVED** [review]
Thanks Nathan!

File: examples/create-database.jl (hunk 3 above)
Comment (NHDaly) [reply examples/create-database.jl:15]: ❤️

File: examples/clone-database.jl (hunk 2 above)
Comment (NHDaly) [reply examples/clone-database.jl:28]: k thanks, i was wondering that too. I agree with you! I'll make that change. 👍

Comment (NHDaly) [reply examples/clone-database.jl]: done!
